src/statement_generator.cpp

```diff
@@ -4,22 +4,30 @@
 #include "duckdb/common/random_engine.hpp"
 #include "duckdb/common/types/uuid.hpp"
 #include "duckdb/function/table/system_functions.hpp"
+#include "duckdb/main/attached_database.hpp"
+#include "duckdb/main/database_manager.hpp"
 #include "duckdb/parser/expression/list.hpp"
+#include "duckdb/parser/parsed_data/create_function_info.hpp"
 #include "duckdb/parser/parsed_data/create_schema_info.hpp"
 #include "duckdb/parser/parsed_data/create_table_info.hpp"
-#include "duckdb/parser/parsed_data/create_view_info.hpp"
-#include "duckdb/parser/parsed_data/create_function_info.hpp"
 #include "duckdb/parser/parsed_data/create_type_info.hpp"
+#include "duckdb/parser/parsed_data/create_view_info.hpp"
 #include "duckdb/parser/parsed_expression_iterator.hpp"
 #include "duckdb/parser/query_node/select_node.hpp"
 #include "duckdb/parser/query_node/set_operation_node.hpp"
+#include "duckdb/parser/statement/attach_statement.hpp"
 #include "duckdb/parser/statement/create_statement.hpp"
 #include "duckdb/parser/statement/delete_statement.hpp"
+#include "duckdb/parser/statement/detach_statement.hpp"
 #include "duckdb/parser/statement/insert_statement.hpp"
+#include "duckdb/parser/statement/multi_statement.hpp"
 #include "duckdb/parser/statement/select_statement.hpp"
+#include "duckdb/parser/statement/set_statement.hpp"
 #include "duckdb/parser/statement/update_statement.hpp"
 #include "duckdb/parser/tableref/list.hpp"
 
+#define TESTING_DIRECTORY_NAME "duckdb_unittest_tempdir"
```

Comment:
Now that duckdb_sqlsmith is an extension and has duckdb as a submodule is it possible to use the `TESTING_DIRECTORY_NAME` declaration from `test/helpers/test_helpers.cpp`?

Reply:
you might have to include `duckdb/test/include/test_helpers.hpp`

Reply:
In GenerateAttachInfo() it checks if directory exists and creates it if it doesn't. Maybe better to use a different name for sqlsmiths testing directory? Something like sqlsmith_unittest_tempdir maybe?

Reply:
Defined new TESTING_DIRECTORY_NAME, also put it in the .gitignore